src/BizHawk.Client.EmuHawk/Program.cs

```diff
@@ -31,8 +31,13 @@ internal static class Program
 		static Program()
 		{
 			// This needs to be done before the warnings/errors show up
+			

```

Comment:
Blank lines here and elsewhere are presumably holdouts from debugging? If this feature is accepted, you should remove whitespace changes.